client/ayon_core/plugins/publish/extract_thumbnail.py

```diff
@@ -303,6 +303,8 @@ def _is_review_instance(self, instance):
         #   other way then checking for "review" key on instance data?
         if instance.data.get("review", True):
             return True
+        if instance.data.get("need_thumbnail", False):
+            return True
```

Comment:
I wonder if we now should refactor the method name to `_is_thumbnail_instance` too to avoid confusion?